packages/calcite-components/src/components/fab/fab.scss

```diff
@@ -1,17 +1,40 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-fab-background-color: Specifies the component's background color.
+ * @prop --calcite-fab-border-color: Specifies the component's border color.
+ * @prop --calcite-fab-corner-radius: Specifies the component's corner radius.
+ * @prop --calcite-fab-text-color: Specifies the component's text color.
+ * @prop --calcite-fab-loader-color: Specifies the component's loader color.
+ * @prop --calcite-fab-shadow: Specifies the component's shadow.
+ */
+
 :host {
-  @apply flex bg-transparent;
+  @apply flex;
+
+  background-color: transparent;
 }
 
 @include disabled();
 
 calcite-button {
-  @apply shadow-2;
+  --calcite-fab-shadow-internal: 0 6px 20px -4px rgba(0, 0, 0, 0.1), 0 4px 12px -2px rgba(0, 0, 0, 0.08);
+  box-shadow: var(--calcite-fab-shadow, var(--calcite-fab-shadow-internal));
+
   &:hover {
     @apply shadow-2-lg;
```

Comment:
Q: can the user customize the shadow when hovered/active using `--calcite-fab-shadow`

Reply:
The shadow for `hover` and `active` is going away and designers advised not to add tokens for these in the time being.